src/App.tsx

```diff
@@ -5,7 +5,11 @@ import { FindMovie } from './components/FindMovie';
 import { Movie } from './types/Movie';
 
 export const App = () => {
-  const [movies] = useState<Movie[]>([]);
+  const [movies, setMovies] = useState<Movie[]>([]);
+
+  const hasMovie = (imdbId: string) => {
+    return !!movies.find(movie => movie.imdbId === imdbId);
```

Comment:
looks like you've implemented `some` method here 😄

Reply:
and let's call it like the function should be called. I'd make it like `checkIsMovieFound`